ReservationService/Controllers/ReservationController.cs

```diff
@@ -176,11 +176,18 @@ private int AmountOfWeekendDays(DateTime startDate, DateTime endDate)
         /// Get all reservations that are linked to a product
         /// </summary>
         /// <param name="productId">productId used to find the reservations</param>
+        /// <param name="excludeHistory">Whether past reservation should be excluded from the results</param>
         /// <returns>All found reservations</returns>
-        [HttpGet("{productId}")]
-        public async Task<ActionResult<IEnumerable<Reservation>>> GetReservationsByProductId(int productId)
+        [HttpGet("{productId}/{excludeHistory}")]
```

Comment:
Dit zou denk ik veel beter een query parameter kunnen zijn, dan hoef je denk ik ook geen changes in ocelot te maken (denk ik?)

Dus dan krijg je ipv:
localhost/api/reservation/1/true
localhost/api/reservation/1?excludeHistory=true